tests/test_cpp_api/test_estimation.cpp

```diff
@@ -19,6 +19,160 @@ class EstimationTest : public KDBTest, public ::testing::Test
 };
 
 
+TEST_F(EstimationTest, SetBlock)
```

Comment:
@jmpplan Could you review this unittest (SetBlock) ? 
The mehod `set_bloc()` is supposed to mimic the behavior of `ODE_blk_check()` but in which I manually pass the values for the 'block' and for the name of the currently displayed equation.

You may have suggestions for more "use cases".